### PR TITLE
Add ability for multiple operators per node

### PR DESCRIPTION
**Pull request by bvbfan**
It starts a thread per master node operator to stake.

### CODE REVIEW

**Comment by monstrobishi** [comment]
Quick question @bvbfan will this affect consensus? I assume no but just checking because it touches some parts of code that have consensus

**Comment by bvbfan** [comment]
Should not affect consensus.

File: src/init.cpp [hunk 1]
```diff
@@ -1915,73 +1915,89 @@ bool AppInitMain(InitInterfaces& interfaces)
...
-                    }
-                }
-                if (!found) {
-                    LogPrintf("Error: masternode operator private key not found\n");
-                    return false;
+        std::set<std::string> operatorsSet;
+        bool atLeastOneRunningOperator = false;
+        auto const operators = gArgs.GetArgs("-masternode_operator");
+
+        for (auto const & op : operators) {
```
Comment (monstrobishi) [inline src/init.cpp:1928]: How does the setting of multiple operators look like in the conf now?

Can we have backwards compatibility with the current setting?

Comment (bvbfan) [reply src/init.cpp:1928]: Backward compatibility is shown in functional test. The settings is same as cli settings you just specify multiple options
`defid -masternode_operator=<fist-one> -masternode_operator=<second-one>`

**Comment by ShengguangXiao** [comment]
So the masternode_owner parameter has no use already?

**Comment by bvbfan** [comment]
It should be used with `-gen` it's not been used till now in tests, probably used in `AmIOperator`

**Review by ShengguangXiao — APPROVED** [review]

**Pull request merged**